decentraland/unity-explorer
# fix: tweaked cursor values so they properly align with real cursor

## What does this PR change?

The issue was that our cursors did not align properly with the real cursor, that is, the expected hotspot of interaction did not match between the hardware cursor and any of our cursors.
In close proximity to the colliders, this wasnt really noticeable, but the further away we went, a few pixels mean we either click on thing or another, specially when there are many colliders on screen.
So here I fixed both the "normal" cursors and the UI Toolkit cursor used for the Crosshair.
This caused for the raycasts to be generated from the "wrong" (from player's perspective) position.


OLD BEHAVIOUR:
https://github.com/user-attachments/assets/438d87f2-60aa-49c1-9463-836232ed1cec

You can see with the crosshair that vertically it is off, the cursor is aligned with the top instead of the center of the crosshair.
And with the arrow and finger pointing cursors, both are off by a few pixels, which means that at bigger distances the error margin looks bigger and bigger.

NEW BEHAVIOUR:
https://github.com/user-attachments/assets/7a92e7ae-a76c-44b0-952d-4229b68ae906


## How to test the changes?

Easiest way is to go to goerli (`/goto goerli`), for example, scene 80,-1 and play with the cube like the in the videos.
The behaviour should be like in the second one, instead of what happend in the first.

Other than that, we can make sure that stuff like the things described in this ticket are not happening anymore, this is more noticeable when we got many avatars in a scene (although as shown in the video below, it also is noticeable with just one avatar as well)
https://github.com/decentraland/unity-explorer/issues/2359

OLD BEHAVIOUR:
https://github.com/user-attachments/assets/dde55499-ff5b-4737-a210-c19d7764cf38

NEW BEHAVIOUR:
https://github.com/user-attachments/assets/97c73d9e-8b09-4785-af77-4fb2c5929495
